test/test_cdd.cpp

```diff
@@ -32,10 +33,26 @@ using std::cerr;
 using std::cout;
 using base::Timer;
 
-/** Random range, may change definition */
-#define RANGE()      ((rand() % 10000) + 1)
-#define RANDOMINT(n) ((rand() % n))
-#define RANDOMBOOL() (rand() > (RAND_MAX / 2))
+/** Random helper functions. */
```

Comment:
Maybe it's not a good idea to reinitialize the random number generator every time it's called? We could also just use the random number generation included in UUtils: https://github.com/UPPAALModelChecker/UUtils/blob/main/include/base/random.h

Reply:
Thanks for the reference to UUtils. Let me see what I can do with that.